ci-operator/step-registry/upi/install/libvirt/upi-install-libvirt-ref.yaml

```diff
@@ -16,6 +16,9 @@ ref:
   - name: DOMAIN_VCPUS
     documentation: Number of VCPUs to allocate for each guest
     default: '6'
+  - name: INSTALLER_TYPE
+    documentation: The type of installer to use for deployment. Supported values are [agent, assisted, base]
+    default: 'base'
```

Comment:
I think I'd prefer `[agent, assisted, default]` as a `base` install-type sounds like it refers to something that would be done as a minimal install or as a first step before doing something more with the installation.  I think its misleading.  (references in if-blocks above would need updated)